Packs/CommonPlaybooks/TestPlaybooks/playbook-Context_Polling_-_DT_-_Test.yml

```diff
@@ -0,0 +1,476 @@
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 0
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: test question
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Test Data Collection
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: cc275f38-f5b5-4f1a-9735-bf7f16620c0c
+    type: playbook
+    task:
+      id: cc275f38-f5b5-4f1a-9735-bf7f16620c0c
+      version: -1
+      name: Context Polling - DT
+      description: This playbook polls a context value until a specific condition, defined by a DT expression, is met.
+      playbookName: Context Polling - DT
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      eval_dt:
+        simple: .=>val.length == 2
+      frequency:
+        simple: "1"
+      playbook_id:
+        simple: ${currentPlaybookID}
+      tags:
+        simple: test-data-collection-task
+      timeout:
+        simple: "100"
+      value_dt:
+        simple: Test Data Collection.Answers
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 910,
+          "y": 870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 88c6c8f4-ce73-41f1-ac50-49aeaf0a6294
+    type: title
+    task:
+      id: 88c6c8f4-ce73-41f1-ac50-49aeaf0a6294
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 695,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1050,
+        "width": 1670,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
+tests:
+- Endpoint Enrichment - Generic v2.1 - Test
```

Comment:
The `tests` field in this test playbook lists numerous unrelated test playbooks. This field should either be empty or only contain `Context Polling - DT - Test` if it's meant for self-testing. Please remove the unrelated entries.